src/App.js

```diff
@@ -1,23 +1,60 @@
-import logo from './logo.svg';
 import './App.css';
+import { useState } from 'react';
+import Parade from './Components/Parade';
+import Sign from './Components/Sign';
 
+// App(): sets up & renders initial state for `isOpen`, `unicornSize`, `lionSize`, and `animals`
+// App(): `isOpen`, `unicornSize`, `lionSize`, and `animals all change on button clicks 
 function App() {
+  // const [isOpen, setIsOpen] = useState('open');
+  const [unicornSize, setUnicornSize] = useState(10);
+  const [lionSize, setLionSize] = useState(10);
+  const [parade, setParade] = useState(['cheetah', 'elephant', 'zebra', 'gorilla']);
+  const [isOpen, setIsOpen] = useState(false);
+
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <div className="fight">
+        <div className="animal">
+          <img src="unicorn.png" width={10 * unicornSize} />
+          {/* the width of the unicorn should be ten times whatever the unicorn size is in state */}
+          <div className="buttons">
+            {/* when you click this button, the unicorn's size in state should go up by one */}
+            <button onClick={() => setUnicornSize(unicornSize + 1)}>The unicorn is growing</button>
+            {/* when you click this button, the unicorn's size in state should go down by one */}
+            <button onClick={() => setUnicornSize(unicornSize - 1)}>The unicorn is shrinking</button>
+          </div>
+          <div className="animal">
+            <img src="lion.png" width={10 * lionSize} />
+            {/* the width of the lion should be ten times whatever the lion size is in state */}
+            <div className="buttons">
+              {/* when you click this button, the lion's size in state should go up by one */}
+              <button onClick={() => setLionSize(lionSize + 1)}>The lion is growing</button>
+              {/* when you click this button, the lion's size in state should go down by one */}
+              <button onClick={() => setLionSize(lionSize - 1)}>The lion is shrinking</button>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div>
+        <Sign isOpen={isOpen} />
+        <div>
+          <button onClick={() => setIsOpen(false)}>Open the Zoo</button>
+          <button onClick={() => setIsOpen(true)}>Close the Zoo</button>
```

Comment:
You have your true and false swapped on lines 42 and 43.  setIsOpen to true to open the zoo, and false to close the zoo.

Reply:
Thank you for pointing this out!